src/device-registry/models/Event.js

```diff
@@ -175,6 +175,18 @@ const valueSchema = new Schema({
       default: null,
     },
   },
+  average_pm2_5: {
+    value: {
+      type: Number,
+      default: null,
+    },
+  },
+  average_pm10: {
+    value: {
+      type: Number,
+      default: null,
+    },
+  },
```

Comment:
**average_pm2_5** and **average_pm10** must  have the same structure as **pm2_5** and **pm10** resp @Baalmart